build-tools/packages/build-tools/src/common/fluidRepo.ts

```diff
@@ -159,6 +159,11 @@ export interface PolicyConfig {
 	handlerExclusions?: { [rule: string]: string[] };
 
 	packageNames?: PackageNamePolicyConfig;
+
+	/**
+	 * (optional) requirements to enforce against each public package.
+	 */
+	publicPackageRequirements?: PackageRequirements;
```

Comment:
I suggest making the corresponding changes to the root fluidbuild config file. We don't validate against a schema or anything like that so the config will just be ignored by earlier versions of the build-tools.

Reply:
Done :) I also ran the fixer and have checked in a fair number of script updates.